### PR TITLE
Update developer guide

### PR DESCRIPTION
**Pull request by yhtMinceraft1010X**
Fixes related typos carried over from AB3 DG: [see this link](https://github.com/nus-cs2103-AY2021S2/forum/issues/273)

Also updates readme and index pages to have new Ui photos as well as remove AB3 tutorials: [see this link](https://github.com/nus-cs2103-AY2021S2/forum/issues/274)

### CODE REVIEW

**Review by figo2127 — COMMENTED** [review]
Thanks for removing them mate! LGTM.

**Review by AiwassPrime — COMMENTED** [review]
LGTM

File: docs/diagrams/ModelClassDiagram.puml [hunk 1]
```diff
@@ -51,7 +51,6 @@ Task *--> "1" Description
 Task *--> "1" Workload
 Task *--> "1" DoneStatus
 Task *--> "*" Tag
-Module ..> Workload
```
Comment (QY-H00) [inline docs/diagrams/ModelClassDiagram.puml:54]: Nice removement for it!

**Review by QY-H00 — COMMENTED** [review]
Overall LGTM.

**Comment by umergta** [comment]
LGTM

**Pull request merged**